src/scenic/syntax/scenic.gram

```diff
@@ -2153,6 +2155,10 @@ scenic_abort_stmt: "abort" { s.Abort(LOCATIONS) }
 
 scenic_take_stmt: "take" elts=(','.expression+) { s.Take(elts=elts, LOCATIONS) }
 
+scenic_wait_for_stmt: "wait" "for" u=scenic_dynamic_duration { s.WaitFor(duration=u, LOCATIONS) }
+
+scenic_wait_until_stmt: "wait" "until" cond=expression { s.WaitUntil(cond=cond, LOCATIONS) }
```

Comment:
```suggestion
scenic_wait_for_stmt: "wait" 'for' u=scenic_dynamic_duration { s.WaitFor(duration=u, LOCATIONS) }

scenic_wait_until_stmt: "wait" 'until' cond=expression { s.WaitUntil(cond=cond, LOCATIONS) }
```
Since `for` and `until` are hard keywords, they should be in single quotes. I guess Pegen is being permissive in also allowing them to appear in double quotes, but we should be consistent.